app/javascript/listings/__tests__/ContactViaConnect.test.jsx

```diff
@@ -16,26 +16,20 @@ describe('<ContactViaConnect />', () => {
 
   it('should render a checked check box opting in to open DMs', () => {
     const onChange = jest.fn();
-    const { queryByLabelText } = render(
+    const { getByText } = render(
       <ContactViaConnect onChange={onChange} checked />,
     );
 
-    expect(
-      queryByLabelText(
-        'Allow Users to Message Me Via In-App Chat (DEV Connect)',
-      ),
-    ).toBeDefined();
+    expect(getByText('Allow Users to message me via Connect.')).toBeDefined();
   });
 
   it('should fire a change event when clicking the checkbox', () => {
     const onChange = jest.fn();
-    const { getByLabelText } = render(
+    const { getByText } = render(
       <ContactViaConnect onChange={onChange} checked />,
     );
 
-    const checkbox = getByLabelText(
-      'Allow Users to Message Me Via In-App Chat (DEV Connect)',
-    );
+    const checkbox = getByText('Allow Users to message me via Connect.');
 
     fireEvent.input(checkbox, { target: { checked: true } });
```

Comment:
I saw the HTML was a bit different. In practice it works is the same way, but with the new structure the query doesn't return the element for some reason (or the event doesn't fire correctly). This worked for me to make `yarn test` pass locally though:

```suggestion
    fireEvent.click(checkbox);
```